### PR TITLE
Spring Web + Smallrye OpenAPI coverage

### PR DESCRIPTION
**Pull request by jsmrcka**

### CODE REVIEW

**Comment by rsvoboda** [comment]
603-spring-web-smallrye-openapi failed

File: 603-spring-web-smallrye-openapi/src/test/java/org/acme/spring/web/OpenApiTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package org.acme.spring.web;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.response.Response;
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.*;
```
Comment (rsvoboda) [inline 603-spring-web-smallrye-openapi/src/test/java/org/acme/spring/web/OpenApiTest.java:6]: remove `*` import

File: 603-spring-web-smallrye-openapi/src/main/resources/META-INF/resources/index.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
```
Comment (rsvoboda) [inline 603-spring-web-smallrye-openapi/src/main/resources/META-INF/resources/index.html:1]: this file is not needed

File: 603-spring-web-smallrye-openapi/src/test/java/org/acme/spring/web/OpenApiTest.java (hunk 1 above)
Comment (rsvoboda) [reply 603-spring-web-smallrye-openapi/src/test/java/org/acme/spring/web/OpenApiTest.java:6]: there 5 occurrences ...

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: 603-spring-web-smallrye-openapi/src/main/resources/META-INF/resources/index.html (hunk 2 above)
Comment (jsmrcka) [reply 603-spring-web-smallrye-openapi/src/main/resources/META-INF/resources/index.html]: Deleted.

File: 603-spring-web-smallrye-openapi/src/test/java/org/acme/spring/web/OpenApiTest.java (hunk 1 above)
Comment (jsmrcka) [reply 603-spring-web-smallrye-openapi/src/test/java/org/acme/spring/web/OpenApiTest.java]: Fixed. I used the Quarkus formatting: https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md#idea-setup
Hope that's ok.

**Comment by jsmrcka** [comment]
603 will keep failing at least until https://github.com/quarkusio/quarkus/issues/15443 gets resolved.
So, I guess we wait with merging until then?

**Comment by rsvoboda** [comment]
Disable the tests in question with link to the issue, we want this module in 1.11 branch too.

**Review by rsvoboda — APPROVED** [review]

**Comment by rsvoboda** [comment]
Failed module is for issue which is just being addressed upstream, not related to this PR.

**Pull request merged**